python/ray/data/_internal/logging_json.yaml

```diff
@@ -0,0 +1,34 @@
+version: 1
```

Comment:
There is a decent amount of shared YAML between this file and logging.yaml, is there a sane way to share that in some base YAML file and or is it reasonable to maintain these two separate files?

Reply:
AFAIK there is no way to construct variables/constants in YAML, and definitely not shared across files